packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -182,66 +181,78 @@ export default function SubmitArticleModal({
   }
 
   return (
-    <ResponsiveModal
-      {...modalProps}
-      onRequestClose={onRequestClose}
-      padding={false}
-      style={{ content: { maxWidth: '35rem', width: '100%' } }}
-    >
-      <header className="flex justify-between items-center px-6 w-full h-14 border-b border-theme-divider-tertiary">
-        <div className="flex flex-row items-center">
-          <h3 className="font-bold typo-title3">{headerCopy}</h3>
+    <form ref={submitFormRef} aria-busy={isValidating} onSubmit={onSubmit}>
+      <Modal
+        {...modalProps}
+        kind={Modal.Kind.FlexibleCenter}
+        size={Modal.Size.Medium}
+        onRequestClose={onRequestClose}
+      >
+        <Modal.Header>
+          <Modal.Header.Title>{headerCopy}</Modal.Header.Title>
           <BetaBadge />
```

Comment:
We can remove this, after discussion on Slack:
https://dailydotdev.slack.com/archives/C02AA18RD1D/p1668588454326879

Reply:
Done 👍